### ISSUE TITLE
No resource identifier found for attribute 'showAsAction' in package 'za.co.neilson.alarm'

### ISSUE THREAD

**Issue by giamby3000** [SheldonNeilson/Android-Alarm-Clock#4]
Hi. 
I download your project (great work!) but in eclipse or Android studio I obtain this message error:

No resource identifier found for attribute 'showAsAction' in package 'za.co.neilson.alarm'

I see you use appcompat:showAsAction="always" and I added v7-appcompaq package into workspace, but it doens't resolve the issue. 

Any idea?

Thank you very much

**Comment by SheldonNeilson** [comment]
Did you add appcompat to the build path or just copy it into the folder? What is your projects's min Android version & build version?

**Comment by giamby3000** [comment]
Hi. 
I imported the app compat in Android Private library (sdk path).
Now I import the compaq project directly in the workspace and now it works
thanks

**Comment by giamby3000** [comment]
In Android Studio after porting from Eclipse I get this error:

Error:(17, -1) android-apt-compiler: [Alarm Clock] C:\Users\GBC\Desktop\a\Alarm Clock\res\menu\menu.xml:17: error: No resource identifier found for attribute 'showAsAction' in package 'za.co.neilson.alarm'

Android Studio is the last one

Any idea?

**Comment by SheldonNeilson** [comment]
Looks like the same issue, only now in Android Studio instead of Eclipse. Ensure that the appcompat library is included in the build path. This article explains how to do it in both environments https://developer.android.com/tools/support-library/setup.html. You may need to modify your build.gradle file.

**Comment by giamby3000** [comment]
Hi. sordry for my long absence.
I fix my problems in Eclipse and in Android studio.
I have another question about your service: I noticed that alarms go off at once then after 24hours thet don't go off at all, if my phone isn't turn off.
any iideas? 
This behaviour occurs with Android 4.2,2, while all is ok in Android 2.3.6. 
I tried to add some instructions at your service destroy method: 

public void onDestroy() {

    super.onDestroy();
    startService(new Intent(this, YourService.class));

}


because I tohught that it was Android that kill your service, but the problem isn't fix

Thanks
